fftoml/fftoml.go

```diff
@@ -6,27 +6,69 @@ import (
 	"io"
 	"strconv"
 
-	"github.com/BurntSushi/toml"
+	"github.com/pelletier/go-toml"
 	"github.com/peterbourgon/ff"
 )
 
 // Parser is a parser for TOML file format. Flags and their values are read
 // from the key/value pairs defined in the config file.
 func Parser(r io.Reader, set func(name, value string) error) error {
-	var m map[string]interface{}
-	_, err := toml.DecodeReader(r, &m)
-	if err != nil {
-		return ParseError{Inner: err}
+	return ParserWith()(r, set)
+}
+
+type config struct {
+	separator string
+}
+
+// Option is a function which configures a fftoml.ConfigFileParser
```

Comment:
Please make sure all prefixing doc comments are full sentences with punctuation. In this case, add a period :)

```suggestion
// Option is a function which changes the behavior of the TOML config file parser.
```